.github/actions/build-frontend/action.yml

```diff
@@ -28,7 +28,7 @@ runs:
     - name: Use Node.js ${{ matrix.node-version }} with yarn
       uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c
       with:
-        node-version: "16.19.1"
+        node-version: "18.15"
```

Comment:
Just checking/belaboring the point -- is it not possible to read this from `engines` or the .nvmrc file?

Reply:
Just waiting on approval from devops on going that route.